<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensor Reader</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>616</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Graph" name="acc" native="true"/>
       </item>
       <item>
        <widget class="Graph" name="gyro" native="true"/>
       </item>
       <item>
        <widget class="Graph" name="velocity" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Graph" name="vLen" native="true"/>
       </item>
       <item>
        <widget class="Graph" name="pos" native="true"/>
       </item>
       <item>
        <widget class="Graph" name="posLen" native="true"/>
       </item>
       <item>
        <widget class="Graph" name="mag" native="true"/>
       </item>
       <item>
        <widget class="Graph" name="pressure" native="true"/>
       </item>
       <item>
        <widget class="Graph" name="temp" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Sensor name</string>
       </property>
       <property name="buddy">
        <cstring>sensorNameBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sensorNameBox"/>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>&amp;Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="handshakeCheckBox">
       <property name="text">
        <string>&amp;Handshaking</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectButton">
       <property name="text">
        <string>&amp;Disconnect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startReadButton">
       <property name="text">
        <string>&amp;Start reading</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopReadButton">
       <property name="text">
        <string>S&amp;top reading</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetReadButton">
       <property name="text">
        <string>&amp;Reset reading</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="readCardButton">
       <property name="text">
        <string>R&amp;ead SD Card</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="analyzeFileButton">
       <property name="text">
        <string>&amp;Analyze file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showFileButton">
       <property name="text">
        <string>Show &amp;file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ti&amp;me scale</string>
       </property>
       <property name="buddy">
        <cstring>xScaleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="xScaleSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Time &amp;grid width[sec]</string>
       </property>
       <property name="buddy">
        <cstring>timeGridSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="timeGridSpinBox">
       <property name="minimum">
        <double>0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&amp;Min graph height</string>
       </property>
       <property name="buddy">
        <cstring>minHeightBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minHeightBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>110</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="syncX">
       <property name="text">
        <string>S&amp;ync slider</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="updateHeadCheckBox">
       <property name="text">
        <string>&amp;Update head</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>&amp;Print graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pdfButton">
       <property name="text">
        <string>Save &amp;PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Graph</class>
   <extends>QWidget</extends>
   <header>graph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
